x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/prebuilt_rules_install_update_workflows.mdx

```diff
@@ -0,0 +1,180 @@
+# Rule Immutability / Customization
```

Comment:
I think this should be renamed to something like "Prebuilt Rules install and update workflows".

While the epic is called "Rule Immutability / Customization", this is the test plan that describes how the feature works, and the feature we are testing is actually the installation and upgrade workflows. Wether rules are immutable or customisable is actually just an implementation feature.

In fact, as part of this milestone, customization is still not possible.

Reply:
As you say, the epic is called "Rule Immutability / Customization" and I think that should also be reflected in the Test Plan. How about we add "Prebuilt Rules install and update workflows" as a subtitle instead of "Test Plan for 2nd Milestone of Customizing prebuilt detection rules"?

Reply:
Synced with @vgomez-el: update title to "Prebuilt Rules install and update workflows" but make clear that this test plan belongs to "Rule Immutability / Customization" epic with the link to the epic ticket.